.github/ISSUE_TRIAGE.md

```diff
@@ -6,44 +6,44 @@ This document describes the triage process used for the [odata.net repository].
 
 We use customer feedback as our primary input for prioritizing issues.
 
-* P1: 5 or more asks, legal or security issues
+* P1: 5 or more asks, legal, or security issues
 * P2: 3-4 asks, regressions
 * P3: 2 asks
 * P4: 1 ask
 
-## What constitutes an ask
+## What constitutes an "ask"
 
-We do our best to interpret customer feedback as asks. The easiest thing for us to interpret is opening an issue or an explicit comment on the issue stating that the fix would be helpful to a different party. We also make a best effort to interpret positive reactions as asks, although this is more subjective.
+We do our best to interpret customer feedback as asks. The easiest thing for us to interpret is opening an issue or an explicit comment on the issue stating that the fix would be helpful to a different party. We also make a best effort to interpret positive reactions as asks, although this interpretation is more subjective.
 
-Duplicate issues also constitute an ask. You can help us identify duplicate issues by adding a comment to the issue that references the duplicate issue number.
+Duplicate issues also constitute an "ask". You can help us identify duplicate issues by adding a comment to the issue that references the duplicate issue number.
 
-Finally, we also count asks from partner teams at Microsoft. We strongly encourage our colleagues to post the ask directly on GitHub, however that doesn't always happen.
+Finally, we also count asks from partner teams at Microsoft. We strongly encourage our colleagues to post the "ask" directly on GitHub, however that doesn't always happen.
 
-## Flagging regressions, legal or security issues
+## Flagging regressions, legal, or security issues
 
-If an issue is a regression from a previous release, please explicitly describe the issue as a regression so that we triage it appropriately. For legal or security issues, please do not report the issue on GitHub but rather send us an e-mail at odatafeedback@microsoft.com.
+If an issue is a regression from a previous release, explicitly describe the issue as a regression so that we triage it appropriately. For legal or security issues, do not report the issue on GitHub but rather send us an e-mail at odatafeedback@microsoft.com.
 
 ## Questions
 
-In general we ask for questions to be posted on Stack Overflow rather than on GitHub. We will tag issues that are actually questions with the question tag. In some cases we may answer the question directly, however we appreciate it the community handles the questions on a forum dedicated to answering questions.
+In general, we ask for questions to be posted on Stack Overflow rather than on GitHub. We tag issues that are actually questions with the question tag. In some cases we may answer the question directly, however we appreciate it the community handles the questions on a forum dedicated to answering questions.

```

Comment:
"In some cases we may answer the question directly, however we appreciate it the community handles the questions on a forum dedicated to answering questions."

Just realized that the sentence is missing a word.

--> "In some cases we may answer the question directly, **though** we appreciate it **if** the community handles the questions on a forum dedicated to answering questions."